src/FadeIn.js

```diff
@@ -44,7 +44,7 @@ export default class FadeIn extends Component {
               className={this.props.childClassName}
               style={{
                 transition: `opacity ${transitionDuration}ms, transform ${transitionDuration}ms`,
-                transform: `translateY(${this.state.maxIsVisible > i ? 0 : 20}px)`,
+                transform: this.state.maxIsVisible ? 'none' : `translateY(${this.state.maxIsVisible > i ? 0 : 20}px)`,
```

Comment:
Does this work with more than one child element? I think as soon as `state.maxIsVisible` changes from `0` to `1`, all child elements will be assigned a `transform: none` rule, which isn't what we want.